tests/unit/test_utils.py

```
@@ -334,3 +337,237 @@ def test_get_default_headers_immutable(self):
+        """Test that json_output=False returns a non-empty string."""
+        utils_module.configs.json_output = False
+        result = format_response(self.PATIENT)
+        assert isinstance(result, str)
+        assert len(result) > 0
+
+    def test_toon_output_is_not_json(self):
+        """Test that toon output is not valid JSON."""
+        import json
+        utils_module.configs.json_output = False
+        result = format_response(self.PATIENT)
+        with pytest.raises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)